### PR TITLE
zet 0.2.0 (new formula)

### PR DESCRIPTION
**Pull request by apcamargo**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

### CODE REVIEW

File: Formula/zet.rb [hunk 1]
```diff
@@ -0,0 +1,21 @@
+class Zet < Formula
+  desc "CLI utility to find the union, intersection, and set difference of files"
+  homepage "https://github.com/yarrow/zet"
+  url "https://github.com/yarrow/zet/archive/refs/tags/0.2.0.tar.gz"
+  sha256 "b001632ecff545411908a9b365dbac6f930e563233547a4cb0ad210d3066952b"
```
Comment (carlocab) [inline Formula/zet.rb:5]: Please include the `license`.

File: Formula/zet.rb [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", "--root", prefix
```
Comment (carlocab) [inline Formula/zet.rb:10]:
```suggestion
    system "cargo", "install", *std_cargo_args
```

**Review by carlocab — COMMENTED** [review]
Thanks, @apcamargo. Just a few suggestions.

**Comment by apcamargo** [comment]
Thanks @carlocab. This software has a dual license (https://github.com/yarrow/zet). What do you recommend doing here?

**Comment by carlocab** [comment]
Use
```ruby
license any_of: ["Apache-2.0", "MIT"]
```

File: Formula/zet.rb [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+  depends_on "rust" => :build
+
+  def install
+    system "cargo", "install", *std_cargo_args
+  end
+
+  test do
+    (testpath/"foo.txt").write("1\n2\n3\n4\n5\n")
+    (testpath/"bar.txt").write("1\n2\n4\n")
+    resource("testdata").stage do
```
Comment (carlocab) [inline Formula/zet.rb:17]: You're trying to use a `resource` that you never declare, so the test is failing.

File: Formula/zet.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
...
+      assert_equal "3/n5/n",
+      shell_output("#{bin}/zet diff foo.txt bar.txt")
+    end
```
Comment (carlocab) [inline Formula/zet.rb:20]:
```suggestion
    assert_equal "3/n5/n", shell_output("#{bin}/zet diff foo.txt bar.txt")
```
I'm not even sure why you need a `resource` here. Doesn't this work?

**Review by dawidd6 — APPROVED** [review]

**Pull request closed without merging**